tools/txg/TXG.hs

```diff
@@ -332,6 +334,26 @@ listenerRequestKey config host listenerRequest = do
     cid :: ChainId
     cid = fromMaybe (unsafeChainId 0) . listToMaybe $ nodeChainIds config
 
+-- | Send a single transaction to the network, and immediately listen for its result.
+singleTransaction :: Args -> HostAddress -> SingleTX -> IO ()
+singleTransaction args host (SingleTX c cid) = do
+  cfg <- mkTXGConfig Nothing args host
+  kps <- testSomeKeyPairs
+  cmd <- mkExec (T.unpack c) (datum kps) (Sim.makeMeta cid) kps Nothing
+  runMaybeT (f cfg cmd) >>= \case
+    Nothing -> putStrLn "Single Transaction: failed" >> exitWith (ExitFailure 1)
```

Comment:
I think there should be some indication as to why the transaction failed. I could imagine someone easily forgetting a parentheses.

Reply:
I'm writing a patch for Pact that will allow this to be cleaner. Notice the `ExceptT`s being hushed into `MaybeT` for now.

Reply:
Cool.

Reply:
You might want to open a new PR addressing this issue. I won't let this issue impede this PR from going through the tubes.